deploy-board/deploy_board/templates/groups/asg_policy.tmpl

```diff
@@ -455,6 +453,159 @@ aria-hidden="true">
+                var scaleDownSteps = config["scaleDownSteps"].trim();
+                var scaleDownAdjustments = config["scaleDownAdjustments"].trim();
+
+                if (scaleUpSteps.length > 0 && scaleUpAdjustments.length == 0) {
+                    alert("Scale up adjustments must be provided for scale up steps!")
+                    return false;
+                }
+
+                if (scaleUpAdjustments.length > 0 && scaleUpSteps.length == 0) {
+                    alert("Scale up adjustments provided, but no scale up step defined!")
+                    return false;
+                }
+
+                if (scaleDownSteps.length > 0 && scaleDownAdjustments.length == 0) {
+                    alert("Scale down adjustments must be provided for scale down steps!")
+                    return false;
+                }
+
+                if (scaleDownAdjustments.length > 0 && scaleDownSteps.length == 0) {
+                    alert("Scale down adjustments provided, but no scale down step defined!")
+                    return false;
+                }
+
+                if (scaleUpSteps.length > 0) {
+                    var scaleUpStepsArray = validateAndParseSteps(scaleUpSteps, true, true)
+                    var scaleUpAdjustmentsArray = validateAndParseSteps(scaleUpAdjustments, true, false)
+
+                    if (scaleUpStepsArray == null || scaleUpAdjustmentsArray == null) {
+                        return false;
+                    }
+
+                    if (scaleUpStepsArray.length != scaleUpAdjustmentsArray.length) {
+                        alert("Each step must have an adjustment defined: " + scaleUpStepsArray + " vs. " + scaleUpAdjustmentsArray)
+                        return false;
+                    } 
+                }
+
+                if (scaleDownSteps.length > 0) {
+                    var scaleDownStepsArray = validateAndParseSteps(scaleDownSteps, false, true)
+                    var scaleDownAdjustmentsArray = validateAndParseSteps(scaleDownAdjustments, false, false)
+
+                    if (scaleDownStepsArray == null || scaleDownAdjustmentsArray == null) {
+                        return false;
+                    }
+
+                    if (scaleDownStepsArray.length != scaleDownAdjustmentsArray.length) {
+                        alert("Each step must have an adjustment defined: " + scaleUpStepsArray + " vs. " + scaleUpAdjustmentsArray)
+                        return false;
+                    } 
+                }
+            }
+
+            function validateAndParseSteps(stepString, isUp, isStep) {
+                if (stepString.length == 0) {
+                    alert("Steps or adjustments cannot be empty");
+                    return null;
+                }
+
+                var stepArray = stepString.split(',').map(Number);
+                for (i = 0; i < stepArray.length; i++) {
+                    if (isNaN(stepArray[i])) {
```

Comment:
nit: can this be Number.isNaN instead? I think the latter is preferred for some type conversion pitfalls w/ the global method, [for ref](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN#description).